libs/shared/methodologies/bold/testing/src/builders/bold-stub.builder.ts

```diff
@@ -213,11 +250,16 @@ export class BoldStubsBuilder {
     return this;
   }
 
-  createParticipantHomologationDocuments(): BoldStubsBuilder {
+  createParticipantHomologationDocuments(
+    homologationDocuments?: Map<
+      (typeof ACTOR_PARTICIPANTS)[number],
+      StubBoldDocumentParameters
+    >,
+  ): BoldStubsBuilder {
```

Comment:
Just a reflexion (no action needed): when I read the term `builder`, it comes the my mind the [builder design pattern](https://refactoring.guru/design-patterns/builder), which is not what we are using here.